Client/Application/AppDelegate.swift

```diff
@@ -33,7 +34,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         let defaultRequest = NSURLRequest(URL: AppConstants.AboutHomeURL)
         let tabManager = TabManager(defaultNewTabRequest: defaultRequest, prefs: profile.prefs)
-
+        self.tabManager = tabManager
```

Comment:
You can drop the local instance on line 36 and just immediately do `tabManager = TabManager(...)`